bin/cloud-push

```diff
@@ -17,20 +17,22 @@ cd "$(dirname "$0")/.."
 
 . misc/shlib/shlib.bash
 
-if [[ $# -lt 1 ]]; then
-    die "usage: $0 DOCKER-HUB-USERNAME [arguments for mzimage]"
+if [[ $# -lt 2 ]]; then
+    die "usage: $0 DOCKER-HUB-USERNAME TAG [arguments for mzimage]"
 fi
 username=$1
 shift
+tag=$1
+shift
 
 for image in environmentd clusterd; do
     bin/mzimage acquire --arch=x86_64 "$image" "$@"
-    docker tag "$(bin/mzimage spec --arch=x86_64 "$image" "$@")" "$username/$image"
-    docker push "$username/$image"
+    docker tag "$(bin/mzimage spec --arch=x86_64 "$image" "$@")" "$username/$image:$tag"
+    docker push "$username/$image:$tag"
 done
 
 echo "Deploy this build to staging by running:"
 echo
 echo "    $ bin/mz profile init --endpoint=https://staging.cloud.materialize.com --profile=staging"
-echo "    $ bin/mz region enable --version $username:latest --region=aws/us-east-1 --profile=staging"
```

Comment:
We need to maintain the `latest` tag on DockerHub because the feature benchmark attempts to fetch the `latest` container and benchmark against it. If DockerHub is not going to do that for us internally, we need to keep tagging `latest` ourselves.

Reply:
oh wait, this is an `echo` . So this script will not do the `latest` tagging. If so, who/where it is being done?

Reply:
Are we using `bin/cloud-push` to upload the latest containers? Before, it would always upload the images to the `latest` tag, without any control, and I made that explicit in this PR. So any place where we used `bin/cloud-push docker-username` before would have to be changed to `bin/cloud-push docker-username latest` now.

The `latest` magic was implicit in `docker tag` and `docker push`, which I change in this script to take an explicit tag.

Reply:
The fact that `latest` is the implicit default tag is documented here: https://docs.docker.com/engine/reference/commandline/tag/

Reply:
I found where it is done by the CI, in `./ci/deploy_mz/docker.py` so this PR will have no impact. Thanks.